src/types.jl

```diff
@@ -147,6 +178,23 @@ struct Bounds{T <: Union{LLA, ENU}}
     max_x::Float64
 end
 
+"""
+    Bounds(min_lat, max_lat, min_lon, max_lon)
+
+Throw an error if the lattitude (`min_lat` and `max_lat`) and longitude (`min_lon` and `max_lon`) bounds are out of range.
+
+# Examples
+```julia-repl
+julia> OpenStreetMapX.Bounds(min_lat, max_lat, min_lon, max_lon)
```

Comment:
this example is not useful as this is literally the same as documentation. Find another one or remove examples section